examples/logical_interconnect_groups.go

```diff
@@ -52,10 +47,10 @@ func main() {
 	logicalLocation1 := ov.LogicalLocation{LocationEntries: *locationEntries1}
 	logicalLocation2 := ov.LogicalLocation{LocationEntries: *locationEntries2}
 	interconnectMapEntryTemplate1 := ov.InterconnectMapEntryTemplate{LogicalLocation: logicalLocation1,
-		PermittedInterconnectTypeUri: "/rest/interconnect-types/f4437afa-529f-467a-9875-7dd3a4fe561f",
+		PermittedInterconnectTypeUri: "/rest/interconnect-types/18f00f2f-505b-4497-a825-8336996196a6",
```

Comment:
Can we get the uri using GetLogicalInterconnectByUri instead of hardcoding which will help us for test automation.

Reply:
done